dist/images/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN yum install -y  \
 	firewalld-filesystem \
 	libpcap \
 	hostname \
+	kubernetes-client \
 	iproute strace socat && \
```

Comment:
@pecameron @dcbw @shettyg If we do a rolling update of the K8s cluster to 1.12.X (for example), then will we have to now re-build the OVN daemonset image since the embedded kubernetes-client may not work with the new K8s version?